index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Portifolio | Stanley</title>
+</head>
+<body>
+  <!--Navigation bar--> 
+    <header> 
+     <div class= "header-logo">
+      <div class="bar-card" id="myNavbar">
+        <a href="#home" class="logo">Ezeoke Stanley</a> 
+  <!--Right-sided navbar links-->
+        <nav>
+        <div class="right-bar">
+          <img src="./icons/menu.png" alt="envelope icon" class="envelope">
+        </div>
+      </nav>
+      </div>
+    </div>
+  </header>
+         <!--Header-->
+         <section class="container">
+           <div class="about">
+             <h1>Hey There.<br>
+              I'm Stan</h1>
+             <h3 class="H-title">I'am a Software Developer</h3>
+             <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+              don’t hestiate to contact me.</p>
+           </div>
+              <!--Social media icon-->
+              <div class="social-media">
+                  <ul>
+                    <li><img
+                        src="icons/github.png"
+                        class="icon"
+                        alt="github icon"
+                        id="github"
+                      />
+                    </li>
```

Comment:
[ Optional Changes ]

- [ ] Kindly place all the image icons in an `a` tag individually, as these are meant to be your social media links that when clicked, will take the person to that particular social media option clicked.
```
<li>
   <a href="#yourgithublink">
      <img
          src="icons/github.png"
          class="icon"
          alt="github icon"
          id="github"
        />
    </a>
</li>
```
What do you think? I'm sure you get the point now right 👍